.github/workflows/build.yml

```diff
@@ -63,7 +63,8 @@ jobs:
           sleep 1
           # We check for broken links:
           # * ignore fonts.gstatic.com which is mentioned by a link preconnect tag that linkchecker does not parse correctly.
-          linkchecker --check-extern --no-robots --ignore fonts.gstatic.com http://localhost:8880/website/ --ignore-url=https://doi.org/10.1002/chem.201803418
+          # * ignore some specific URLs that work fine in the browser but reject bots.
+          linkchecker --check-extern --no-robots --ignore fonts.gstatic.com --ignore-url='https://doi.org/10.1002/chem.201803418' http://localhost:8880/website/
```

Comment:
I have no idea why this fixes the linkchecker but it seems to work.

Reply:
I don't think this fixes anything, it just randomly works now.

Anyway, it's maybe a little bit better documented now, so we probably want to merge this.

Reply:
I also tried a plethora of different options and it never really worked except with this approach...
I agree, its better when it is documented.